components/uploadthing-ui/button-generic-drive.tsx

```
@@ -0,0 +1,523 @@
+  fileId,
+  file,
+  status,
+  abortSignal,
+  onStatusChange,
+  UTUIFunctionsProps,
+}: FileUploaderProps) {
+  // [1] State & Ref
+  const [progress, setProgress] = useState(0);
+  const isMounted = useRef(true);
+  const hasStartedUpload = useRef(false);
+  const {} = useGenericDriveStore();
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)